unison-syntax/src/Unison/Syntax/Lexer.hs

```diff
@@ -889,7 +889,10 @@ lexemes' eof =
           <|> layoutKeywords
       where
         keywords =
-          symbolyKw "."
+          -- yes "wordy" - just like a wordy keyword like "true", the literal "." (as in the dot in
+          -- "forall a. a -> a") is considered the keyword "." so long as it is either followed by EOF, a space, or some
+          -- non-wordy character (because ".foo" is a single identifier lexeme)
+          wordyKw "."
```

Comment:
Why not `symboly`, because the "eof/space/non-wordy-character" definition of wordy just _is_ what we need here?

Reply:
Yeah, the definition of `wordyKw` is what we need here. I can't remember exactly what bug `symbolyKw` was causing at the moment, unfortunately, but it's not correct